config/config.go

```diff
@@ -37,6 +37,8 @@ import (
 var (
 	_evmNetworkID uint32
 	loadChainID   sync.Once
+	_chainID      uint32
+	_onceChainID  sync.Once
```

Comment:
we can now get chainID from blockchainCtx, which comes from genesis.Config that was used to start the node
so I think we don't need to add these 2